how-to/integrate-with-sso/integrate-with-auth0/client/src/auth.ts

```diff
@@ -259,31 +293,50 @@ async function authenticatedStateChanged(
 	pollTimerStart(isAuthenticated);
 }
 
-function pollTimerStop() {
+/**
+ * Stop the token verification polling timer.
+ */
+function pollTimerStop(): void {
 	if (pollTimerId) {
 		window.clearInterval(pollTimerId);
 		pollTimerId = undefined;
 	}
 }
 
-function pollTimerStart(isAuthenticated: boolean) {
-	if ((authSettings.verifyPollMs ?? 0) > 0 && isAuthenticated) {
+/**
+ * Start the token verification polling timer.
+ * @param isAuthenticated Is the user authenticated.
+ */
+function pollTimerStart(isAuthenticated: boolean): void {
```

Comment:
I think there is a new API in container to know when a page has navigated without the polling.

Reply:
Actually this timer is a background process which checks for the token still being valid, that being said I have updated the other timer for the auth window to use the `url-changed` event.